csc_335_restaurant/src/model/Server.java

```diff
@@ -8,56 +8,47 @@
 package model;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 
 public class Server {
-	// private ArrayList<Table> tables;
+	private ArrayList<Integer> tables;
 	private String name;
-	private int num_seated;
 	private double tips;
 	
 	public Server(String name) {
-		//tables = new ArrayList<Table>();
+		this.tables = new ArrayList<Integer>();
 		this.name = name;
-		num_seated = 0;
-		tips = 0.0;
+		this.tips = 0.0;
 	}
 	
-	/*
-	public void seatTable(Tabel t) {
-		// when every table is sat num_seated is updated to add the num of seats from that table
-		tables.add(t);
-		num_seated += t.getSeats();
+	public void addTable(int table) {
+		tables.add(table);
 	}
 	
-	public Table popTable(Table t) {
-		// when every table is popped num_seated is updated to remove the num of seats from that table
-		num_seated -= t.getSeats();
-		return tables.remove(t);
+	public void removeTable(int table) {
+		tables.remove(table);
```

Comment:
The reason is because remove has two `remove(...)` methods:

- `remove(int index)`
→ Removes element at position index
→ Example: `list.remove(3)` removes the 4th item

- `remove(Object o)`
→ Removes the first occurrence of object o
→ Example: `list.remove(Integer.valueOf(3))` removes the first element equal to 3
```suggestion
		tables.remove(Integer.valueOf(table));
```